README.md

```diff
@@ -68,3 +68,11 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+# Screenshots
+
+[1st person position recognition](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-51-03.jpg)
+[recognition of the position of several people at once](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-51-03%20(2).jpg)
+[architecture resNet50](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-56-46.jpg)
+[architecture mobileNetV1](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-57-10.jpg)
```

Comment:
The interface is already included in the main screenshot. I think you can remove these ones

Reply:
> 

I suggest you consider the option that the screenshots below are displayed in this file. Did I do the right thing by creating a folder with screenshots in the root of the project?

Reply:
please check readme.md. for correctness